### PR TITLE
bumps docusaurus to 2.0 (stable)

### PR DESCRIPTION
**Pull request by pruthvi2103**
## Description

Bumps docusaurus to version 2.0

## Motivation and Context

Always better to work with a stable release of a library

## How Has This Been Tested?

Tested in dev env

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by pruthvi2103** [comment]
closes #61

File: lib/package.json [hunk 1]
```diff
@@ -24,7 +24,9 @@
     "@vanilla-extract/sprinkles": "^1.3.3",
     "@vanilla-extract/vite-plugin": "^3.1.0",
     "deepmerge": "^4.2.2",
-    "polished": "^4.1.3"
+    "polished": "^4.1.3",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0"
```
Comment (Raalzz) [inline lib/package.json:29]: Can we make this, above or equal to 17?
```suggestion
    "react": ">=17.0.0",
    "react-dom": ">=17.0.0"
```

Ref:
https://nodejs.org/en/blog/npm/peer-dependencies/
image

Comment (pruthvi2103) [reply lib/package.json:29]: sure we can do this, as of today, i do not see any issue

**Review by Raalzz — APPROVED** [review]
LGTM

**Pull request merged**